docs/_dg/UseCases.md

```diff
@@ -166,9 +166,9 @@ Extensions:
 
 #### Use Case 8: Removing a tag from an item
 
-Preconditions: The tag to be removed is tagged to the item currently.
+Preconditions: The [[ tag ]] to be removed is tagged to the item currently.
 
-MSS:
+[[ MSS ]]:
```

Comment:
Like OS, this is an invalid key as the glossary entry name is "MSS (Main Success Scenario)". However, I'm going to change this to "Main Success Scenario (MSS)" in another PR, so let's fix it the right way.

```suggestion
[[ main-success-scenario-mss:MSS ]]:
```